ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -50,55 +50,44 @@ def __init__(self):
         self.loop()
 
     def loop(self):
-        rate = rospy.Rate(5) # 5Hz
+        rate = rospy.Rate(25) # 25 Hz is half of the twist controller
```

Comment:
Rate of 25Hz, seems to be too frequent to me. Since the LOOKAHEAD_WPS is 100, which is approximately 100 meters ahead of ego car as I calculated, 5Hz seems to be enough for speed at around 10m/s. Increasing this value will hurt the computational load, so please reconsider.

Reply:
Yes 5 is good. I wanted to do some test on the change of performance but I had no time.